test/WebSites/FilesWebSite/Controllers/DownloadFilesController.cs

```diff
@@ -23,12 +25,28 @@ public IActionResult DownloadFromDisk()
             return PhysicalFile(path, "text/plain");
         }
 
+        public IActionResult DownloadFromDisk_WithLastModifiedAndEtag()
+        {
+            var path = Path.Combine(_hostingEnvironment.ContentRootPath, "sample.txt");
+            var lastModified = DateTimeOffset.Parse("05/01/2008 +1:00");
```

Comment:
DTO.Parse() uses the current culture, so this will not have the desired behavior on some machines. Either use ParseExact, or call a DTO ctor that takes in explicit year/month/day/time/etc. integers, e.g. https://msdn.microsoft.com/en-us/library/bb354978(v=vs.110).aspx